src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -813,6 +819,20 @@ public void onTraversalStart() {
         );
     }
 
+    /**
+     *  Checks to see if the given vcf has already been annotated.
+     *
+     *  No need to annotate again.
+     */
+    static void checkIfAlreadyAnnotated(final VCFHeader vcfHeader) {
```

Comment:
If this method has package level access so that test code can access it, it should be annotated with the `@VisibleForTesting` annotation.